repose-aggregator/components/filters/openstack-identity-v3-filter/src/test/scala/org/openrepose/filters/openstackidentityv3/OpenStackIdentityV3HandlerDeprecatedTest.scala

```diff
@@ -0,0 +1,677 @@
+/*
+ * _=_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=
+ * Repose
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Copyright (C) 2010 - 2015 Rackspace US, Inc.
+ * _-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_-_=_
+ */
+package org.openrepose.filters.openstackidentityv3
+
+import java.util.{Calendar, GregorianCalendar}
+import javax.servlet.http.HttpServletResponse
+
+import com.mockrunner.mock.web.{MockHttpServletRequest, MockHttpServletResponse}
+import org.junit.runner.RunWith
+import org.mockito.Matchers.{eq => mockitoEq}
+import org.mockito.Mockito.{verify, when}
+import org.openrepose.commons.utils.http.{CommonHttpHeader, HttpDate, OpenStackServiceHeader}
+import org.openrepose.commons.utils.servlet.filter.FilterAction
+import org.openrepose.commons.utils.servlet.http.HttpServletRequestWrapper
+import org.openrepose.filters.openstackidentityv3.config.OpenstackIdentityV3Config.SendProjectIdQuality
+import org.openrepose.filters.openstackidentityv3.config._
+import org.openrepose.filters.openstackidentityv3.objects._
+import org.openrepose.filters.openstackidentityv3.utilities._
+import org.scalatest._
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.mock.MockitoSugar
+import org.springframework.http.HttpHeaders
+
+import scala.util.{Failure, Try}
+
+@RunWith(classOf[JUnitRunner])
+class OpenStackIdentityV3HandlerDeprecatedTest extends FunSpec with BeforeAndAfterEach with Matchers with PrivateMethodTester with MockitoSugar {
```

Comment:
I'm not certain why you elected to duplicate this entire test suite versus just adding your new tests to the other and modifying the few for the now deprecated option.